### PR TITLE
Prebid 7: Remove stored auction response

### PR DESCRIPTION
**Pull request by patmmccann**
Fixes https://github.com/prebid/Prebid.js/issues/8157

### CODE REVIEW

**Review by dgirardi — APPROVED** [review]
LGTM

**Comment by dgirardi** [comment]
Also I'll mention https://github.com/prebid/Prebid.js/issues/7796 for tracking

**Comment by patmmccann** [comment]
@harpere or @robertrmartinez could you help us out with a magnite review for the second review here

File: modules/rubiconBidAdapter.js [hunk 1]
```diff
@@ -311,11 +311,6 @@ export const spec = {
 
       applyFPD(bidRequest, VIDEO, data);
 
-      // if storedAuctionResponse has been set, pass SRID
```
Comment (bretg) [inline modules/rubiconBidAdapter.js:314]: Actually, I do have a request here. In order to remove this code, I need the adapter to read ortb2Imp and merge it into the PBS request.

The adunit will contain, for example:
```
      ortb2Imp: {
        ext: {
          prebid: {
            storedauctionresponse: { id: "sample_video_response" }
          }
        }
      },
``` 
The video 'fork' of the rubicon adapter needs to pass this on imp[].ext.prebid.storedauctionresponse.id. It doesn't currently.

**Review by bretg — CHANGES_REQUESTED** [review]

File: modules/rubiconBidAdapter.js (hunk 1 above)
Comment (patmmccann) [reply modules/rubiconBidAdapter.js:314]: I'd feel more comfortable leaving this change to a magnite developer; they can feel free to push changes to the branch  in the Pr 'remove-stored-auction-response'

**Comment by patmmccann** [comment]
@bretg any news on a magnite contribution on the magnite adapter part of the pr with the change you requested?

**Review by bretg — APPROVED** [review]

**Review by harpere — APPROVED** [review]
Rubicon Bid Adapter changes LGTM

**Pull request merged**